test/test_automl/test_estimators.py

```diff
@@ -26,650 +28,599 @@
+    )
+    classifier.InputValidator.validate_target(
+        pd.DataFrame(expected_result, dtype='int64'),
+        is_classification=True,
+    )
+
+    actual_result = classifier.predict([None] * len(predicted_indexes))
+
+    np.testing.assert_array_equal(expected_result, actual_result)
+
+
+def test_can_pickle_classifier(tmp_dir, output_dir, dask_client):
+    X_train, Y_train, X_test, Y_test = putil.get_dataset('iris')
+    automl = AutoSklearnClassifier(time_left_for_this_task=30,
+                                   per_run_time_limit=5,
+                                   tmp_folder=tmp_dir,
+                                   dask_client=dask_client,
+                                   output_folder=output_dir)
+    automl.fit(X_train, Y_train)
+
+    initial_predictions = automl.predict(X_test)
+    initial_accuracy = sklearn.metrics.accuracy_score(Y_test,
+                                                      initial_predictions)
+    assert initial_accuracy >= 0.75
+    assert count_succeses(automl.cv_results_) > 0
+
+    # Test pickle
+    dump_file = os.path.join(output_dir, 'automl.dump.pkl')
+
+    with open(dump_file, 'wb') as f:
+        pickle.dump(automl, f)
+
+    with open(dump_file, 'rb') as f:
+        restored_automl = pickle.load(f)
+
+    restored_predictions = restored_automl.predict(X_test)
+    restored_accuracy = sklearn.metrics.accuracy_score(Y_test,
+                                                       restored_predictions)
+    assert restored_accuracy >= 0.75
+    assert initial_accuracy == restored_accuracy
+
+    # Test joblib
+    dump_file = os.path.join(output_dir, 'automl.dump.joblib')
+
+    joblib.dump(automl, dump_file)
+
+    restored_automl = joblib.load(dump_file)
+
+    restored_predictions = restored_automl.predict(X_test)
+    restored_accuracy = sklearn.metrics.accuracy_score(Y_test,
+                                                       restored_predictions)
+    assert restored_accuracy >= 0.75
+    assert initial_accuracy == restored_accuracy
+
+
+def test_multilabel(tmp_dir, output_dir, dask_client):
+
+    X_train, Y_train, X_test, Y_test = putil.get_dataset(
+        'iris', make_multilabel=True)
+    automl = AutoSklearnClassifier(time_left_for_this_task=30,
+                                   per_run_time_limit=5,
+                                   tmp_folder=tmp_dir,
+                                   dask_client=dask_client,
+                                   output_folder=output_dir)
+
+    automl.fit(X_train, Y_train)
+    # Log file path
+    log_file_path = glob.glob(os.path.join(
+        tmp_dir, 'AutoML*.log'))[0]
+    predictions = automl.predict(X_test)
+    assert predictions.shape == (50, 3), extract_msg_from_log(log_file_path)
+    assert count_succeses(automl.cv_results_) > 0,  extract_msg_from_log(log_file_path)
+
+    score = f1_macro(Y_test, predictions)
+    assert score >= 0.9, extract_msg_from_log(log_file_path)
 
-        pickle.dumps(automl_fitted)
```

Comment:
Hey, I think something went wrong during the merge or rebase, and the test that this can be pickled is now missing.

Reply:
thanks for catching it, I have fixed this.